### DISCUSSION TITLE
Gittyup locking up

### DISCUSSION THREAD

**Discussion by ghost** [Murmele/Gittyup#678] [General]
Gittyup is locking up on me as I scroll and click on commits. I tried first on Linux and then on Windows and both have the same problem.
Downloaded the dev version and still have the problem. I even tried going back to a previous version. Must be something to do with repo as it quite old and large.

**Comment by XeonG** [comment]
gittyup definitely has a problem with large number of files or binary files or something.. I can't figure out specific thing causes it.. but easier for me to stage and commit files in terminal than even bother using the gui of this as it just dies with large uncommitted or commits that contain a lot of changes and showing the files..... only tried smartgit and it doesn't have the same problem so is some bug or design feature that just ain't working right.
